db/wal_dump.cc

```diff
@@ -0,0 +1,39 @@
+#include "rocksdb/wal_dump.h"
+#include <memory>
+#include <thread>
+
+#include "rocksdb/env.h"
+#include "db/wal_manager.h"
+#include "db/version_set.h"
+#include "options/db_options.h"
```

Comment:
Not intended to be exposed.

Reply:
Does rocksdb provide similar functional API?